AUTHORS

```diff
@@ -0,0 +1,24 @@
+Chip Senkbeil <chip.senkbeil@gmail.com>
+Dale Noe <dalenoe@gmail.com>
+Gabor Nagy <Aigeruth@users.noreply.github.com>
+Harry Khanna <harry@khanna.cc>
+Ihor Dvoretskyi <ihor@linux.com>
+Jacob Wood <jacoblukewood@gmail.com>
+Jan Baudisch <dev@baudisch.xyz>
+Jiri Hubacek <jiri.hubacek@gmail.com>
+Khosrow Moossavi <khos2ow@gmail.com>
+Luiz de Prá <luiz.pra@olist.com>
+Luiz de Prá <rawarkanis@gmail.com>
+Luiz F. A. de Prá <luizdepra@users.noreply.github.com>
+Luiz F. A. de Prá <rawarkanis@gmail.com>
```

Comment:
I appear here 4 times. ahahahaha
But I don't need to be listed in this file.

Reply:
To me this's fine.